docs/user/metacat/source/intro.rst

```diff
@@ -58,43 +58,40 @@ please see http://knb.ecoinformatics.org.
 
```

Comment:
oracle link (line 57) should be removed, since we now only support postgres

Reply:
line 53: Java Servlets are no known as Jakarta Servlets, apparently 🤷 - We should re-label, and use the new link, in case the redirect disappears over time. (http://en.wikipedia.org/wiki/Java_Servlet redirects to https://en.wikipedia.org/wiki/Jakarta_Servlet)

Reply:
Thank you! I have updated this section.